graphql/mutation_resolver.go

```diff
@@ -401,7 +401,7 @@ func (r *mutationResolver) AttachProjectToRepo(ctx context.Context, projectID st
 }
 
 // CreateProject is the resolver for the createProject field.
-func (r *mutationResolver) CreateProject(ctx context.Context, project restModel.APIProjectRef) (*restModel.APIProjectRef, error) {
+func (r *mutationResolver) CreateProject(ctx context.Context, project restModel.APIProjectRef, createAws *bool) (*restModel.APIProjectRef, error) {
```

Comment:
Should we add some message to the user indicating that this ticket was created? Or that AWS access was requested and it will take some time? Maybe to the success toast ?(this may need to be a front end change)